internal/providers/terraform/aws/efs_file_system.go

```diff
@@ -145,3 +125,39 @@ func calculateProvisionedThroughput(gbStorage *decimal.Decimal, throughput decim
 
 	return &decimal.Zero
 }
+func StorageStandardCostComponent(name, region, usagetype string, quantity *decimal.Decimal) *schema.CostComponent {
+	return &schema.CostComponent{
+
+		Name:            name,
+		Unit:            "GB",
+		UnitMultiplier:  1,
+		MonthlyQuantity: quantity,
+		ProductFilter: &schema.ProductFilter{
+			VendorName:    strPtr("aws"),
+			Region:        strPtr(region),
+			Service:       strPtr("AmazonEFS"),
+			ProductFamily: strPtr("Storage"),
+			AttributeFilters: []*schema.AttributeFilter{
+				{Key: "usagetype", ValueRegex: strPtr(fmt.Sprintf("/%s/i", usagetype))},
+			},
+		},
+	}
+}
+func StorageOneZoneCostComponent(name, region, usagetype string, quantity *decimal.Decimal) *schema.CostComponent {
```

Comment:
Looks like this function are equals to `StorageStandardCostComponent`, so I think you can remove it and use just `efsStorageCostComponent` for example

Reply:
Done